pluggy.py

```diff
@@ -581,7 +595,7 @@ def varnames(func):
         try:
             func = func.__init__
         except AttributeError:
-            return ()
+            return (), ()
```

Comment:
@RonnyPfannschmidt @nicoddemus Afaik this isn't "really" part of the public API due to the contents of `pluggy.__all__`?
Do we know of projects which are using it as such? [`pytest` doesn't seem to be](https://github.com/pytest-dev/pytest/search?utf8=%E2%9C%93&q=varnames)?

I think it's a bit of a leap to assume that `varnames` is *public* just because it's on of the few functions in `pluggy` that doesn't have a leading `_` ;)
Imho we should be able to define functions with normal names and assume they're for internal use *unless* exposed through `__all__`.
That being said I can add the wrapper but I do think this kind of stuff should start being split up into multiple modules; eg. a `utils` module in this case.

@RonnyPfannschmidt regarding vendoring in pytest I **really** think we should be running the `pytest` test suite as part of our CI process in `pluggy` since it is the major dependent project. 
What do you think about me making an issue for that?
I feel that if we get that in place we can work towards a much tighter release process between the two projects where `pytest` can use latest `pluggy` if available and fail over to the vendored version. This would of course mean that `pytest` **does not** require `pluggy` as a default dependency.